src/utils/signers.ts

```diff
@@ -0,0 +1,28 @@
+import type { ConnectedWallet } from '@/hooks/wallets/useOnboard'
+import type { SafeTransaction } from '@safe-global/safe-core-sdk-types'
+import type { SafeInfo } from '@safe-global/safe-gateway-typescript-sdk'
+
+export const getAvailableSigners = (
```

Comment:
I would suggest testing this function

Reply:
Addressed in 56de0ed1f7cc052f870738d6184a6b7e77981ca1.